src/main/java/seedu/contax/ui/ResultDisplay.java

```diff
@@ -2,27 +2,44 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.List;
+
 import javafx.fxml.FXML;
-import javafx.scene.control.TextArea;
+import javafx.geometry.Insets;
 import javafx.scene.layout.Region;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
 
 /**
- * A ui for the status bar that is displayed at the header of the application.
+ * Ui for the status bar that is displayed at the header of the application.
  */
 public class ResultDisplay extends UiPart<Region> {
 
     private static final String FXML = "ResultDisplay.fxml";
 
     @FXML
-    private TextArea resultDisplay;
+    private TextFlow resultDisplay;
 
+    /**
+     * Constructor for ResultDisplay UI component
+     * Used to set initial properties of resultDisplay TextFlow
+     */
     public ResultDisplay() {
         super(FXML);
+        resultDisplay.setPadding(new Insets(5, 10, 5, 10));
+
+        //Code to make the text not wrap, wip to make it scroll
+        resultDisplay.setPrefWidth(Region.USE_COMPUTED_SIZE);
+        resultDisplay.setMinWidth(Region.USE_PREF_SIZE);
     }
 
+
     public void setFeedbackToUser(String feedbackToUser) {
         requireNonNull(feedbackToUser);
-        resultDisplay.setText(feedbackToUser);
+        resultDisplay.getChildren().clear();
+        List<Text> generateStyledText = TextStyleHelper.formattedTextParser(feedbackToUser);
+        for (int i = 0; i < generateStyledText.size(); i++) {
+            resultDisplay.getChildren().add(generateStyledText.get(i));
+        }
```

Comment:
Consider simplifying the code here.
```suggestion
        List<Text> generatedStyledText = TextStyleHelper.formattedTextParser(feedbackToUser);
        resultDisplay.getChildren().setAll(generatedStyledText);
```

Reply:
thanks for the catch, resolved in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/207/commits/5712c55b1563e8adfdcf92abef182a1197e32964